xbmc/windowing/android/WinSystemAndroidGLESContext.h

```diff
@@ -41,6 +41,10 @@ class CWinSystemAndroidGLESContext : public CWinSystemAndroid, public CRenderSys
   bool IsHDRDisplay() override;
   bool SetHDR(const VideoPicture* videoPicture) override;
 
+  bool DisplaySupportsHLG() override;
```

Comment:
Pending my other comment.

These overrides shouldn't be needed as this class derives CWinSystemAndroid which derives CWinSystem.

Reply:
Indeed they're unnecessary, I've removed them.